<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1442</width>
    <height>891</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="DrawPanel" name="drawpanel" native="true">
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>20</y>
            <width>171</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Ağaçları beyaza boyayın.</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="eraserButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>100</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Ağaç (S)</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>140</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Şekiller</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="circleButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>170</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Daire (D)</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="rectangleButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>200</y>
            <width>121</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Dikdörtgen (F)</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="triangleButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Üçgen (G)</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="FillWithColorButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>260</y>
            <width>171</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Şeklin içini doldur (H)</string>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QCheckBox" name="showOriginalButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>300</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Orijinal resmi göster
(boşluk)</string>
          </property>
         </widget>
         <widget class="Line" name="line_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>330</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QToolButton" name="erosionButton">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>350</y>
            <width>81</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>erosion (E)</string>
          </property>
         </widget>
         <widget class="QToolButton" name="dilationButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>350</y>
            <width>91</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>dilation (W)</string>
          </property>
         </widget>
         <widget class="QToolButton" name="resetToAutoButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>380</y>
            <width>181</width>
            <height>61</height>
           </rect>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhMultiLine</set>
          </property>
          <property name="text">
           <string>reset to the 
 auto-generated mask (R)</string>
          </property>
         </widget>
         <widget class="QToolButton" name="finishButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>580</y>
            <width>181</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>finish </string>
          </property>
         </widget>
         <widget class="QSlider" name="penSizeButton_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>160</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="Line" name="line_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>60</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Kalem boyutu</string>
          </property>
         </widget>
         <widget class="QToolButton" name="undoButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>450</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>undo (Ctrl+Z)</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>480</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>(undo only works once)</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>520</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>zoom in: scroll up</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>540</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>zoom out: scroll down</string>
          </property>
         </widget>
         <widget class="Line" name="line_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>510</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="Line" name="line_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>560</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Customize</string>
    </property>
    <addaction name="actionPen_size"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuDraw">
     <property name="title">
      <string>Draw...</string>
     </property>
     <addaction name="actionCircle"/>
     <addaction name="actionRectangle"/>
     <addaction name="actionTriangle"/>
    </widget>
    <addaction name="menuDraw"/>
    <addaction name="separator"/>
    <addaction name="actionFill_with_coor"/>
    <addaction name="actionEraser"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Original_Image"/>
    <addaction name="separator"/>
    <addaction name="actionDilation"/>
    <addaction name="actionErosion"/>
    <addaction name="actionFinish_2"/>
    <addaction name="actionReset_to_the_Auto_Generated_Mask"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuAdd"/>
  </widget>
  <action name="actionPen_size">
   <property name="text">
    <string>Pen size</string>
   </property>
  </action>
  <action name="actionFill_with_coor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fill with color</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="actionZoom_2">
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionShow_Original_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Original Image</string>
   </property>
  </action>
  <action name="actionReset_to_the_Auto_Generated_Mask">
   <property name="text">
    <string>Reset to the first mask (R)</string>
   </property>
   <property name="toolTip">
    <string>Reset to the Auto-Generated Mask (R)</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionDilation">
   <property name="text">
    <string>Dilation (W)</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionErosion">
   <property name="text">
    <string>Erosion (E)</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionFinish_2">
   <property name="text">
    <string>Finish (Enter)</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionUndo_Ctrl_Z">
   <property name="text">
    <string>Undo (Ctrl+Z)</string>
   </property>
   <property name="toolTip">
    <string>Undo (Ctrl+Z) (it works only once)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawPanel</class>
   <extends>QWidget</extends>
   <header location="global">drawpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>eraserButton</sender>
   <signal>clicked()</signal>
   <receiver>actionEraser</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>circleButton</sender>
   <signal>clicked()</signal>
   <receiver>actionCircle</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1108</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rectangleButton</sender>
   <signal>clicked()</signal>
   <receiver>actionRectangle</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1108</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triangleButton</sender>
   <signal>clicked()</signal>
   <receiver>actionTriangle</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1108</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FillWithColorButton</sender>
   <signal>clicked()</signal>
   <receiver>actionFill_with_coor</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1118</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showOriginalButton</sender>
   <signal>clicked()</signal>
   <receiver>actionShow_Original_Image</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1103</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>penSizeButton_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>actionPen_size</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>843</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
